lib/sqlalchemy/testing/reflection_fixture.py

```diff
@@ -0,0 +1,201 @@
+from . import config
+from .assertions import eq_
+from .assertions import is_
+from .assertions import is_not_
+from .assertions import is_true
+from .fixtures import TablesTest
+from .. import inspect
+from .. import Integer
+from ..schema import Column
+from ..schema import Computed
+from ..schema import MetaData
+from ..schema import Table
+
+
+class ComputedReflectionFixtureTest(TablesTest):
```

Comment:
OK this is very nice but I want to see if we can get this to work out with the existing multi-dialect testing system, because otherwise we're inventing a new system, and also I would want this suite to automatically come online for any third party dialect that opts into it using the standard means.

so this would mean:

1. it's in lib/sqlalchemy/testing/suite/test_reflection.py as another "componentreflection" suite, it can be its own class since this is pretty different.

2. The flags that are set, like support_stored, support_virtual, etc., this kind of thing usually comes either from the dialect itself, e.g. there are flags on dialects like "supports_returning", or they come from the exclusion rules in lib/sqlalchemy/testing/requirements.py  test/requirements.py.   The exclusion rules are where third party dialects can also set up a matrix of what capabilities and expectations those dialects have; while you've likely seen the exclusion rules used as decorators on tests, they can also be used as boolean flags inside of tests.     so we can add exclusion rules for now like computed_columns_stored, computed_columns_virtual, im not sure if I like that but it will get us closer.

3. for a quick way to deal with the to_sqltext() thing for now, see below.

Reply:
I'll try to refactor it a bit. The problem was that the sqltext is all over the place for the various backends. I'll try to see if a regexp can fix it

Reply:
> So regarding the `reflection_fixture`
> Should the flag come from the dialect or be placed in requirements?

so it's nice when the flags come from the dialect, but usually the dialect has such flags only if they are needed elsewhere, like if the compiler needed to know about it.  if we're talking about two flags, then I think requirements is OK for now